treeherder/etl/bugzilla.py

```diff
@@ -63,7 +63,8 @@ def run(self):
                             'keywords': ",".join(bug['keywords']),
                             'os': bug.get('op_sys', ''),
                             'modified': dateutil.parser.parse(
-                                bug['last_change_time'], ignoretz=True)
+                                bug['last_change_time'], ignoretz=True),
+                            'whiteboard': bug.get('whiteboard', ''),
```

Comment:
We'll need to fetch the max length from the model definition and truncate the string before saving, to prevent errors should we encounter a longer whiteboard value in the future